tests/components/application-connector/Makefile

```diff
@@ -1,45 +1,47 @@
-NAMESPACE ?= "test"
-
-DOCKER_TAG ?= "PR-14743"
-DOCKER_PUSH_REPOSITORY ?= "eu.gcr.io/kyma-project"
+NAMESPACE = test
 GOPATH ?= $(shell go env GOPATH)
 
-GATEWAY_TEST_IMAGE ?= "$(DOCKER_PUSH_REPOSITORY)$(DOCKER_PUSH_DIRECTORY)/gateway-test:$(DOCKER_TAG)"
-MOCK_APP_IMAGE ?= "$(DOCKER_PUSH_REPOSITORY)$(DOCKER_PUSH_DIRECTORY)/mock-app:$(DOCKER_TAG)"
-
-APP_URL = test-app.test.svc.cluster.local
+GATEWAY_TEST_IMAGE = "$(DOCKER_PUSH_REPOSITORY)$(DOCKER_PUSH_DIRECTORY)/gateway-test:$(DOCKER_TAG)"
+MOCK_APP_IMAGE = "$(DOCKER_PUSH_REPOSITORY)$(DOCKER_PUSH_DIRECTORY)/mock-app:$(DOCKER_TAG)"
 
-.PHONY: release test-os test image publish clean image-gateway publish-gateway test-gateway certs image-mock-app publish-mock-app
+.PHONY: release test-os test-skr test image publish clean image-gateway-test publish-gateway-test test-gateway image-mock-app publish-mock-app
 
 test: test-gateway
-image: image-gateway
-clean: clean-gateway
-publish: publish-gateway publish-mock-app
+image: image-gateway-test
+clean: clean-gateway-test
 
-release: publish
+release: publish-gateway-test publish-mock-app
 test-os: test-gateway
 test-skr: test-gateway
 
 test-gateway:
 	@echo $(GOPATH)/bin/go-junit-report --help
 	kubectl create namespace $(NAMESPACE) --dry-run=client -o yaml | kubectl apply -f -
-	helm template k8s/gateway-test --set gatewayTestImage=$(GATEWAY_TEST_IMAGE),mockImage=$(MOCK_APP_IMAGE),namespace=$(NAMESPACE) | kubectl apply -f -
-	sleep 180
-	@echo ""
+	helm template k8s/gateway-test | kubectl apply -f -
```

Comment:
Values in `values.yaml` have to be in sync with values when pushing, removing variables will make it a bit harder to keep those in sync, but that's only a problem for local development, which is not really a priority.